ethereum_history_api/oracles/src/noir/oracles/receiptOracle.test.ts

```diff
@@ -4,13 +4,14 @@ import { decodeGetReceiptArguments } from './receiptOracle.js';
 describe('receiptOracle', () => {
   describe('decodeGetReceiptArguments', () => {
     it('success', () => {
-      expect(decodeGetReceiptArguments([['0xf'], ['0xb4']])).toStrictEqual({
+      expect(decodeGetReceiptArguments([['0x01'], ['0xf'], ['0xb4']])).toStrictEqual({
+        chainId: 1n,
         blockNumber: 15n,
         txId: 180
       });
     });
     it('get_receipt requires 2 arguments', () => {
```

Comment:
You've forgot to change test name here: 

```suggestion
    it('get_receipt requires 3 arguments', () => {
```